.tests/npmplus-logs/parser.assert

```diff
@@ -0,0 +1,174 @@
+results["s01-parse"]["ZoeyVid/npmplus-logs"][0].Success == true
```

Comment:
Within the tests, can we have the full output, including all stages

Reply:
like now?

Reply:
Yes!